tests/test_entities.py

```diff
@@ -442,6 +444,28 @@ def test_os_default_template(self):
                 self.cfg, id=2, operatingsystem=1).path()
         )
 
+    def test_refresh_externalusergroup(self):
```

Comment:
hmm, report still showing that method as not covered:
![image](https://user-images.githubusercontent.com/10895065/51338681-254f9d00-1a93-11e9-83bc-cea64bd5ec61.png)

Reply:
This test verifies `path()`, not `refresh()`, test name is misleading. `refresh()`, however, isn't covered at all.